### DISCUSSION TITLE
Delete Header in Mailtemplate

### DISCUSSION THREAD

**Discussion by silvanhornstein** [Budibase/budibase#8151] [Help] [answered]
Hi Budibase PPl!
I am sending automated mails through Budibase (Self Hosted) and I am trying to get rid of the e-mail header displaying the name of the budibase instanz and linking to it.
However, I cant find any way to accomplish that.
Any advice, or is this a bug?
Best wishes and keep on rocking making budibase such an awesome tool!

**Comment by andz-bb** [comment, marked as answer]
Hey @silvanhornstein 

If you go to Manage > Email on the sidebar of the Budibase builder and then go to the `Base format` template you'll be able to adjust the email header in there. Hope this helps, if you have any further questions on this let me know 👍

**Comment by silvanhornstein** [comment]
I dont know how i missed that. Thanks for your help!
